infrastructure/variables.tf

```diff
@@ -1,4 +1,8 @@
-variable "product" {}
+variable "product" {
+  type        = string
+  description = "The name of your application"
+  default     = "civil-sdt"
```

Comment:
I believe the product variable is set from the product property in the Jenkinsfile.  The value of the product property in both Jenkinsfile_CNP and Jenkins_nightly is "civil", so this default value isn't correct.

Reply:
ha.. good point, yes, removed those now

Reply:
Default value has been changed to "civil".